website/docs/faqs/Models/reference-models-in-another-project.md

```diff
@@ -3,9 +3,19 @@ title: How can I reference models or macros in another project?
 description: "Use packages to add another project to your dbt project"
 sidebar_label: 'Reference models or macros in another project'
 id: reference-models-in-another-project
-
+keywords: 
+  - project dependency, project dependencies, ref project, dbt mesh, multi-project, mesh, cross-project dependencies
 ---
 
-You can use [packages](/docs/build/packages) to add another project to your dbt
-project, including other projects you've created. Check out the [docs](/docs/build/packages)
-for more information!
+In dbt, you can manage dependencies across multiple dbt projects using:
+
+1. **Packages**: You can install [packages](/docs/build/packages) as a way to add another project to your dbt project, including other projects you've created. When you install a project as a package, you bring in its entire source code, making its macros and models available in your own project.
+
+While this is useful for reusing code and sharing macros, it may not be the best approach for large-scale collaboration, especially in larger organizations.
```

Comment:
gotcha, thanks for testing that @joellabes ! I've added the lowercase redirect but i don't think there's any other way around it. I'm going to cc @john-rock @JKarlavige or @breezyfasano in case they have more insight?

Reply:
ok tested it and it worked! thanks @joellabes !

Reply:
Adding a redirect is definitely the quickest fix. I have a task to look into the Vercel URLs to see how we can serve the same page both lower and capital case in URLs. Here's a link to the ticket https://app.asana.com/0/1200099998847559/1205508339405074/f